### PR TITLE
Update mx.m3u

### PR DESCRIPTION
**Pull request by trancer1994**

### CODE REVIEW

File: channels/mx.m3u [hunk 1]
```diff
@@ -55,3 +55,31 @@ http://s5.mexside.net:1935/telemax/telemax/.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Telemax
+http://s5.mexside.net:1935/telemax/telemax/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Multimedios Guadalajara
+http://mdstrm.com/live-stream-playlist_400/5c54d38ca392a5119bb0aa0d.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Multimedios Laguna
+http://mdstrm.com/live-stream-playlist_400/57bf686a61ff39e1085d43e1.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Mexiquense TV
+https://5ca3e84a76d30.streamlock.net/mexiquense/videomexiquense/chunklist_w1045015137.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcQezVfVamzd08ywLqOjqVqfOtEL-ZyphG2b0w&usqp=CAU" group-title="",TELEMUNDO HD MEX
+https://wmaqlive-f.akamaihd.net/i/wmaqb1_1@24420/index_1286_av-p.m3u8?sd=10&rebase=on
```
Comment (freearhey) [inline channels/mx.m3u:81]: The link is broadcasting NBC Chicago, which is American.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked link again.

**Comment by freearhey** [comment]
@trancer1994 Just to clarify: should I close your other PRs or do you plan to edit them later?

**Comment by trancer1994** [comment]
@Freearhy Feel free to close this one in particular if you want as I might not find any other channels, but leave specifically the ES one open as there's a channel Allegria TV which I suspect may be broadcasting from Spain. Also I'll also be editing some other files later with other channels I find in my text document I have which is basically a text version of the unsorted list from about a few days ago. But now I don't know whether to edit it anymore since someone on another thread has basically called me an asshole and accused me of stealing others' hard work, it's that IPTV-free guy again. The trouble is I believe some of the unsorted channels in my list are the same as was in his, so don't know whether to commit them to the individual countries' files here. I don't want to get accused of stealing stuff.

**Comment by freearhey** [comment]
@trancer1994 If this guy has any complaints about our playlists and wants us to remove some channels he owns the rights to, he can write [here](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=DMCA&template=--remove-channel.md&title=Remove%3A+xxx) and we will remove them all within one working day.

**Comment by trancer1994** [comment]
@Freearhey He doesn't own the rights to any channels. He is that same person who made an issue asking you to remove all the links from his playlists claiming this repo stole a bunch of them. Now he's started commenting on the first pull request I made where I updated the UK list for the first time and calling me an arsehole and saying what a hard-working hore I am.

**Comment by freearhey** [comment]
@trancer1994 If they bother you, you can just [block or report](https://docs.github.com/en/communities/maintaining-your-safety-on-github/blocking-a-user-from-your-personal-account) them. In this case they will no longer be able to comment on your work.

**Pull request closed without merging**